assets/src/userSettings.ts

```diff
@@ -25,11 +32,16 @@ export const vehicleLabelSetting = (
     ? settings.shuttleVehicleLabel
     : settings.ladderVehicleLabel
 
-type VehicleLabelData = "run_id" | "vehicle_id"
+export type VehicleLabelData = "run_id" | "vehicle_id"
+export type TripLabelData = "origin" | "destination"
 
 interface SettingsData {
   ladder_page_vehicle_label: VehicleLabelData
   shuttle_page_vehicle_label: VehicleLabelData
+  // We allow minischedules_trip_label to be undefined to handle the
+  // hybrid code problem, we can disallow undefined after the first
+  // deploy of the corresponding backend code.
+  minischedules_trip_label?: TripLabelData
```

Comment:
Ah, this makes me realize that I should do the same in #1035 before merging.

Reply:
Though once both of these are deployed, it's probably worth cleaning up the front-end code.

Reply:
yes we should make tickets to remove these bits at the first convenient opportunity (post-deploy)

Reply:
I created [a ticket](https://app.asana.com/0/1148853526253420/1199999682730453/f).